scripts/gendocs/generate

```diff
@@ -142,23 +144,34 @@ $output
 EOF
 }
 
+# The branches named here exist in the `jetstack/cert-manager` repo.
+
+genversionwithcli "release-1.8" "next-docs"
+genversionwithcli "release-1.7" "docs"
+genversionwithcli "release-1.7" "v1.7-docs"
+
+# Special handling for release-1.6 because the legacy APIs were still present in
+# the CRDs and in public `pkg/` directory but we do not want to generate API
+# docs for those legacy API docs.
+# But we do need the legacy API source files to be available so that the CLI for
+# this version can be compiled to output the `--help` docs
```

Comment:
I added this extra comment, explaining that the following block contains special API and CLI docs generation.